slidez-core/src/main/java/com/binarystudio/academy/slidez/room/state/State.java

```diff
@@ -0,0 +1,28 @@
+package com.binarystudio.academy.slidez.room.state;
+
+import com.binarystudio.academy.slidez.room.exceptions.PollNotFoundException;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+public class State {
```

Comment:
Does this really need to be public? It's not even a snapshot, I don't see much reason to expose `State` to anything but Room module